CHANGELOG.md

```diff
@@ -263,6 +264,7 @@
 - PR #5745 Update JNI cast for inability to cast timestamp and integer types
 - PR #5750 Add RMM_ROOT/include to the spdlog search path in JNI build
 - PR #5763 Update Java slf4j version to match Spark 3.0
+- PR #5816 Always preserve list column hierarchies across operations.
```

Comment:
The extra appearance of this entry here is odd.

Reply:
This was dependent on that PR, which has since been merged.  I'll get everything synced up and updated on this one this afternoon.

Reply:
It's just strange that the merge of the dependent PR did not resolve this.